schema/deploy/mutations/delete_project_revision.sql

```diff
@@ -0,0 +1,10 @@
+-- Deploy cif:mutations/delete_project_revision to pg
+begin;
+create or replace function cif.delete_project_revision(revision_id integer) returns void as $$
+    begin
+        delete from cif.project_revision_amendment_type where project_revision_id = revision_id;
+        delete from cif.project_revision where id = revision_id;
+    end;
+$$ LANGUAGE plpgsql;
+grant execute on function cif.delete_project_revision to cif_internal, cif_external, cif_admin;
+commit;
```

Comment:
This can be done as an `sql` function rather than `plpgsql`. If possible we should always prefer `sql` over `plpgsql` as it has an effect on performance when using the latter.